### ISSUE TITLE
Report better line numbers for Rubocop lints found in HAML filters

### ISSUE THREAD

**Issue by sds** [brigade/haml-lint#75]
Right now any lints found in interpolation within a HAML filter are reported with a line number that is the first line of that filter. It should instead report on the actual line of the problem.

For example:

```haml
:preserve
  #{Module::method} # Rubocop ColonMethodCall reported on line 1 instead of line 2
```

There might be some difficulties and edge cases here (for example interpolation that spans multiple lines) but we should be able to get somewhere with it.
